### PR TITLE
Godep update

### PR DESCRIPTION
**Pull request by whyrusleeping**
updating a few dependencies that have had bugfixes lately

### CODE REVIEW

File: repo/fsrepo/fsrepo.go [hunk 1]
```diff
@@ -420,13 +416,7 @@ func (r *FSRepo) Close() error {
...
 	}
 
-	if err := r.metricsBlocks.Close(); err != nil {
-		return err
-	}
-	if err := r.metricsLevelDB.Close(); err != nil {
-		return err
-	}
-	if err := r.leveldbDS.Close(); err != nil {
+	if err := r.ds.(io.Closer).Close(); err != nil {
```
Comment (jbenet) [inline repo/fsrepo/fsrepo.go]: this will close `metricsBlocks, metricsLevelDB, leveldbDS`, right?

Comment (whyrusleeping) [inline repo/fsrepo/fsrepo.go]: yeap. the mount datastore closes all child datastores if possible.

**Comment by rht** [comment]
(...??? why is there a go-log lurking in Godeps? should I steal the chance to unify logging before @whyrusleeping does?)

**Comment by whyrusleeping** [comment]
@rht i was going to look at using gx to vendor go-datastore, which should fix the problem. although it wont work 'nicely' until we fully support go1.5 for the `-vendor` flag that gx was designed to use.

**Pull request merged**